aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecMediaSpan.kt

```diff
@@ -98,26 +98,26 @@ class AztecMediaSpan(context: Context, drawable: Drawable?, var attributes: Attr
         val sb = StringBuilder()
         sb.append("<")
         sb.append(TAG)
+        sb.append(' ')
 
-        attributes?.let {
-            for (i in 0..attributes!!.length-1) {
-                sb.append(' ')
-                sb.append(attributes!!.getLocalName(i))
-                sb.append("=\"")
-                sb.append(attributes!!.getValue(i))
-                sb.append("\"")
-            }
-        }
+        attributes.removeAttribute("aztec_id")
 
+        sb.append(attributes)
         sb.append("/>")
+
         return sb.toString()
     }
 
     fun onClick(view: View) {
+        // add a unique ID that can identify this image span
```

Comment:
> maybe it should be added to every element, not just images.

That's basically what I'd expect the client to do: to proactively set IDs to elements, if it needs them.

> > Even if we keep the idea of performing this in onClick, the client can do that, if it wishes to, in its onMediaTappedListener?.mediaTapped() implementation.
> 
> As mentioned above, if you don't have an ID already, you won't be able to find the element to add it.

Let me be more clear: Upon click, client's `mediaTapped` is called. The client can perform the "if attribute missing, add a random value to it" operation in that call since the element's attributes are passed to it via a parameter of `mediaTapped`. There is no need to _search_ for the element inside that call, right?